src/components/GhostCard.tsx

```diff
@@ -1,45 +1,40 @@
 import React from "react"
-import styled from "@emotion/styled"
-
-const Container = styled.div`
-  position: relative;
-  margin-top: 0.5rem;
-  align-self: stretch;
-`
-
-const BaseCard = styled.div`
-  height: 100%;
-  width: 100%;
-  border-radius: 2px;
-`
-
-const Card = styled(BaseCard)`
-  z-index: 2;
-  padding: 1.5rem;
-  text-align: left;
-  border: 1px solid ${(props) => props.theme.colors.border};
-  background-color: ${(props) => props.theme.colors.ghostCardBackground};
-`
-
-const Ghost = styled(BaseCard)`
-  z-index: -1;
-  position: absolute;
-  bottom: 0.5rem;
-  left: 0.5rem;
-  border: 1px solid ${(props) => props.theme.colors.border};
-  background-color: ${(props) => props.theme.colors.ghostCardGhost};
-`
+import { Box } from "@chakra-ui/react"
 
 export interface IProps {
   children?: React.ReactNode
   className?: string
 }
 
 const GhostCard: React.FC<IProps> = ({ children, className }) => (
-  <Container className={className}>
-    <Ghost />
-    <Card className="ghost-card-base">{children}</Card>
-  </Container>
+  <Box position="relative" mt="2" alignSelf="stretch" className={className}>
+    <Box
+      zIndex="hide"
+      position="absolute"
+      backgroundColor="ghostCardGhost"
+      bottom="2"
+      left="2"
+      border="1px solid"
+      borderColor="border"
+      borderRadius="2px"
+      height="full"
+      width="full"
```

Comment:
Two things can be improved with these three props.

Since they are used in both child components, maybe they should be in a reusable object and spread into each component?

Secondly, since both the `height` and `width` props in both instances have the same value (`full`), you can use the `boxSize` prop instead!

So in total...
```tsx
// Imported Type signature from Chakra totally optional, but great to help with the intellisense! :D
import { type ChakraProps } from '@chakra-ui/react'

const baseCardStyles: ChakraProps = {
  boxSize: "full",
  borderRadius: "2px"
}

return (
  // ... //
  <Box 
    {...baseStyles}
  // ... //
```

Purely semantics here, but I see this as a cleaner way to go (and utilizing a Chakra-specific prop)! 😅

Reply:
Didn't know about `boxSize` 👍🏼  TIL